src/server/main_service.cc

```diff
@@ -1133,6 +1134,12 @@ void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx)
     if (auto& exec_info = dfly_cntx->conn_state.exec_info; exec_info.IsCollecting())
       exec_info.state = ConnectionState::ExecInfo::EXEC_ERROR;
 
+    // We need to skip this because ACK's should not be replied to
```

Comment:
why did replconf ack return error in VerifyCommandState?

Reply:
*bug* :rofl:

Reply:
bug in our replica?

Reply:
yes when the default user got its `replconf` acl revoked. Similar to the test case I wrote, but for `default` user (now we support replication with any acl user)